staging/src/k8s.io/kubectl/pkg/cmd/testing/util.go

```diff
@@ -178,3 +180,13 @@ func InitTestErrorHandler(t *testing.T) {
 		t.Errorf("Error running command (exit code %d): %s", code, str)
 	})
 }
+
+func WithAlphaEnv(key string, t *testing.T, f func(*testing.T)) {
```

Comment:
Nit: I suggest either adding a comment here, or using a strongly-typed value for key, or renaming to RunWithEnvVar.  I know it's test code, but it's otherwise what this function does is a little obscure (to me, anyway).